src/webview/mod.rs

```diff
@@ -60,6 +60,20 @@ use crate::application::{dpi::PhysicalSize, window::Window};
 
 use http::{Request, Response};
 
+pub struct RequestApi {
```

Comment:
Let's also add documentation about what's this and where will use it.
```suggestion
pub struct Response{
```